tests/test_xpub_manual.cpp

```diff
@@ -458,50 +458,57 @@ void test_user_message ()
 
 void test_user_message_multi ()
 {
+    const int only_first_subscribe = 1;
+
     //  Create a publisher
     void *pub = test_context_socket (ZMQ_XPUB);
     TEST_ASSERT_SUCCESS_ERRNO (zmq_bind (pub, "inproc://soname"));
+    TEST_ASSERT_SUCCESS_ERRNO (zmq_setsockopt (pub, ZMQ_ONLY_FIRST_SUBSCRIBE,
```

Comment:
the test body needs to be wrapper in ```#ifdef ZMQ_ONLY_FIRST_SUBSCRIBE``` since it will be DRAFT for a while

Reply:
@bluca I guess it should the same in the xsub.cpp and xpub.cpp?

Reply:
Should be fixed now.